cmake/Reprojucer.cmake

```diff
@@ -3618,7 +3618,7 @@ function(_FRUT_set_compiler_and_linker_settings target)
           OUTPUT_STRIP_TRAILING_WHITESPACE
         )
         if(IS_DIRECTORY "${sysroot}")
-          target_compile_options(${target} PRIVATE "-isysroot ${sysroot}")
+          target_compile_options(${target} PRIVATE -isysroot ${sysroot})
```

Comment:
@scotchi it is advised to quote paths, in case they contain spaces, or worse semicolons. Please keep double-quotes around `${sysroot}`.